tensorflow/python/ops/resource_variable_ops.py

```diff
@@ -670,10 +670,15 @@ def _map_resources(self, save_options):
     resource_map = {self.handle: new_variable.handle}
     return obj_map, resource_map
 
-  def _read_variable_op(self):
+  def _read_variable_op(self, no_copy = False):
+    """Reads the value of the variable. If the variable is in copy-on-read mode
```

Comment:
The docstring needs to be in the format of the [Google Python Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods) to pass linters.